doc/crypto/api/ops/aead.rst

```diff
@@ -70,6 +70,10 @@ AEAD algorithms
 
     The CCM block cipher mode is defined in :RFC-title:`3610`.
 
+    .. subsection:: Usage in zigbee
```

Comment:
Editorial: “Zigbee” should be capitalized. (It used to be “ZigBee”, but [that stopped years ago](https://en.wikipedia.org/wiki/Talk:Zigbee#Capitalization:_Zigbee,_not_ZigBee); permanently-uncapitalized “zigbee” seems to be the organization logo, not the name of the protocol suite).

Reply:
I've corrected all references except the document citation: the spec itself uses the uncapitalized form for its title and publisher.